xgcm/grid.py

```diff
@@ -1373,23 +1388,30 @@ def get_metric(self, array, axes):
                             # we found a set of metrics with dimensions compatible with the array
                             metric_vars = possible_combinations
```

Comment:
So you prefer to combine the metric before interpolating it? Should the user have an option to switch this order?